torch/csrc/jit/codegen/cuda/compute_at_map.cpp

```diff
@@ -554,15 +554,22 @@ void ComputeAtMap::build(
       int concrete_count = counts.first;
       int broadcast_count = counts.second;
 
+      // If ID is marked as a count_one id, we'd prefer a non-count one id. This
+      // will encourage selecting an rfactor domain
+      bool count_reset =
+          count_one_concrete_dims_.find(id) != count_one_concrete_dims_.end();
+
       if (concrete_count > max_concrete_count) {
         max_concrete_count = concrete_count;
         max_broadcast_count = broadcast_count;
         concrete_id = id;
       } else if (concrete_count == max_concrete_count) {
-        if (broadcast_count > max_broadcast_count) {
+        if (broadcast_count > max_broadcast_count ||
+            (!count_reset && max_count_reset)) {
```

Comment:
Not quite sure how this selects an rfactor ID. `count_reset` for an rfactor ID should be `true`, so `(!count_reset && max_count_reset)` should be always `false`. 

I also don't follow what `max_count_reset` does.